src/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -13,68 +13,145 @@ public class WindowAndCursorProps : RemoteExecutorTestBase
 {
     [Fact]
     [PlatformSpecific(TestPlatforms.AnyUnix)]  // Expected behavior specific to Unix
-    public static void BufferSize_SettingNotSupported()
+    public static void BufferWidth_GetUnix_ReturnsWindowWidth()
     {
-        Assert.Throws<PlatformNotSupportedException>(() => Console.BufferWidth = 1);
-        Assert.Throws<PlatformNotSupportedException>(() => Console.BufferHeight = 1);
+        Assert.Equal(Console.WindowWidth, Console.BufferWidth);
     }
 
     [Fact]
     [PlatformSpecific(TestPlatforms.AnyUnix)]  // Expected behavior specific to Unix
-    public static void BufferSize_GettingSameAsWindowSize()
+    public static void BufferWidth_SetUnix_ThrowsPlatformNotSupportedException()
     {
         Assert.Throws<PlatformNotSupportedException>(() => Console.BufferWidth = 1);
-        Assert.Throws<PlatformNotSupportedException>(() => Console.BufferHeight = 1);
+    }
 
-        Assert.Equal(Console.WindowWidth, Console.BufferWidth);
+    [Fact]
+    [PlatformSpecific(TestPlatforms.AnyUnix)]  // Expected behavior specific to Unix
+    public static void BufferHeight_GetUnix_ReturnsWindowHeight()
+    {
         Assert.Equal(Console.WindowHeight, Console.BufferHeight);
     }
 
     [Fact]
-    [PlatformSpecific(TestPlatforms.Windows)]  // Expected behavior specific to Windows
-    public static void WindowWidth_WindowHeight_InvalidSize()
+    [PlatformSpecific(TestPlatforms.AnyUnix)]  // Expected behavior specific to Unix
+    public static void BufferHeight_SetUnix_ThrowsPlatformNotSupportedException()
+    {
+        Assert.Throws<PlatformNotSupportedException>(() => Console.BufferWidth = 1);
```

Comment:
This duplicates BufferWidth_SetUnix_ThrowsPlatformNotSupportedException etc above?

Reply:
Fixed...